prow/life_of_a_prow_job.md

```diff
@@ -58,7 +58,7 @@ status:
   state: triggered
 ```
 
-Every thirty seconds, `cmd/plank` runs [`Sync`](https://github.com/kubernetes/test-infra/blob/c8829eef589a044126289cb5b4dc8e85db3ea22f/prow/plank/controller.go#L71). `Sync` runs [`syncKubernetesJob`](https://github.com/kubernetes/test-infra/blob/c8829eef589a044126289cb5b4dc8e85db3ea22f/prow/plank/controller.go#L233-L296) for each Prow job and pod. Because the above Prow job lacks a corresponding Kubernetes pod, `Sync` creates one in the [`test-pods`](https://github.com/kubernetes/test-infra/blob/c8829eef589a044126289cb5b4dc8e85db3ea22f/prow/cluster/test_pod_namespace.yaml) namespace. Finally, `Sync` updates the status line on the PR with a link to Gubernator. Gubernator handles real-time logs and results.
+On ProwJob or Pod changes, `cmd/plank` runs [`Sync`](https://github.com/kubernetes/test-infra/blob/c8829eef589a044126289cb5b4dc8e85db3ea22f/prow/plank/controller.go#L71-L106). `Sync` runs [`syncKubernetesJob`](https://github.com/kubernetes/test-infra/blob/c8829eef589a044126289cb5b4dc8e85db3ea22f/prow/plank/controller.go#L233-L296) for each Prow job and pod. Because the above Prow job lacks a corresponding Kubernetes pod, `Sync` creates one in the [`test-pods`](https://github.com/kubernetes/test-infra/blob/c8829eef589a044126289cb5b4dc8e85db3ea22f/prow/cluster/test_pod_namespace.yaml) namespace.
```

Comment:
This links to a revision from 2017, today neither prow/plank/controller.go exists nor does plank have a `syncKubernetesJob` function, as it only syncs kubernetets jobs as jenkins jobs got their own binary.

Maybe just point to https://github.com/kubernetes/test-infra/blob/a94e7c78b4dd3e2a288f2d74aaac6839bd7e5bc4/prow/plank/reconciler.go#L175 ? This is the entrypoint today

Reply:
I am not familiar with the work flow. I added a suggested edit. Is that sufficient? The function is uncommented and it seems I need to do a bit of digging to figure out all the goes on under the hood.